Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm

```diff
@@ -182,6 +184,8 @@
 
 void Download::setPlaceholderURL(NSURL *placeholderURL, NSData *bookmarkData)
 {
+    RELEASE_LOG(Network, "Download::setPlaceholderURL");
```

Comment:
Probably a good idea to add `m_downloadID` to ALL logging in this class, to differentiate multiple downloads.